samza-core/src/test/java/org/apache/samza/clustermanager/TestContainerProcessManager.java

```diff
@@ -71,7 +71,7 @@
       put("cluster-manager.container.retry.count", "1");
       put("cluster-manager.container.retry.window.ms", "1999999999");
       put("cluster-manager.allocator.sleep.ms", "1");
-      put("cluster-manager.container.request.timeout.ms", "2");
+      put("cluster-manager.container.request.timeout.ms", "100");
```

Comment:
Before this patch this config was no op since expiry did not apply to ContainerAllocator without host affinity. Hence I changed this config to fix failing tests.

Reply:
Does this increase the test-time in case of a timeout test?